### PR TITLE
Blue: Add new bidder

### PR DESCRIPTION
**Pull request by ValeriiBarsuk**
Blue: Add new bidder

### CODE REVIEW

File: src/main/resources/bidder-config/generic.yaml [hunk 1]
```diff
@@ -4,6 +4,17 @@ adapters:
     aliases:
       genericAlias:
         enabled: false
+      blue:
+        enabled: false
+        endpoint: https://prebid-us-east-1.getblue.io/?src=prebid
```
Comment (Net-burst) [inline src/main/resources/bidder-config/generic.yaml:9]: This is a regional endpoint. Is this correct?

File: src/main/resources/bidder-config/generic.yaml [hunk 2]
```diff
@@ -4,6 +4,17 @@ adapters:
...
+        meta-info:
+          maintainer-email: prebid@getblue.io
+          site-media-types:
+            - banner
+          app-media-types:
+            - banner
+          supported-vendors:
+          vendor-id: 0
```
Comment (Net-burst) [inline src/main/resources/bidder-config/generic.yaml:17]: @SerhiiNahornyi are we sure that they don't have a GVL id?

File: src/main/resources/bidder-config/generic.yaml (hunk 1 above)
Comment (ValeriiBarsuk) [reply src/main/resources/bidder-config/generic.yaml:9]: Seems like yes

https://github.com/prebid/prebid-server/blob/64c5e35af385b0999e3c534dcb8ebaf7af15d846/static/bidder-info/blue.yaml#L3

File: src/main/resources/bidder-config/generic.yaml (hunk 2 above)
Comment (SerhiiNahornyi) [reply src/main/resources/bidder-config/generic.yaml:17]: According to https://iabeurope.eu/vendor-list-tcf-v2-0/ they have
Value is 620

Comment (SerhiiNahornyi) [reply src/main/resources/bidder-config/generic.yaml:17]: In go https://github.com/prebid/prebid-server/pull/2323/files#diff-2ca656a1264a75707e04fdda5158c8f7fb3bff8c4afa71c3bf2fc729cd9919c8
They also have gvlVendorID with value 620.
@ValeriiBarsuk Pls update

**Pull request merged**